lib/routes/sign.js

```diff
@@ -81,24 +81,24 @@ module.exports = function (log, P, isA, error, signer, db, domain, devices) {
             function () {
               if (publicKey.algorithm === 'RS') {
                 if (! publicKey.n) {
-                  return reply(error.missingRequestParameter('n'))
+                  throw error.missingRequestParameter('n')
```

Comment:
Since this was just returning, later promises in the `then` chain would end up calling `reply` also. Hapi properly only sends 1 response, but then throws an exception at using it more than once. The exception was swallowed by the unhandled rejection. Once that was fixed, these needed to change to throws so we wouldn't call `reply` again.